packages/replay/src/replay.ts

```diff
@@ -830,6 +833,10 @@ export class ReplayContainer implements ReplayContainerInterface {
     }
 
     this._performanceCleanupCallback = setupPerformanceObserver(this);
+
+    const client = getClient();
+    // Start listening for errors that occur in the core SDK
+    client && client.on && client.on('beforeSendEvent', this._handleSentryEvent);
```

Comment:
This might be better here: https://github.com/getsentry/sentry-javascript/blob/develop/packages/replay/src/util/addGlobalListeners.ts#L37 and move `_handleSentryEvent` into a module